cuequivariance_torch/tests/primitives/equivariant_tensor_product_test.py

```diff
@@ -48,8 +45,8 @@ def make_descriptors():
     )
 
     # These ETPs will trigger the symmetricContraction kernel
-    yield descriptors.spherical_harmonics(cue.SO3(1), [0, 1, 2, 3, 4, 5])
```

Comment:
Here I put back a lower degree polynomial.
The degree 5 was giving a bit too high errors compared to all the other ETP tested.